int32 COBID
std_msgs/Int32MultiArray data
float16 period
